collections/ansible_collections/cloudkit/service/plugins/filter/find_template_roles.py

```diff
@@ -188,6 +188,12 @@ def serialize_path(self, value: Path):
     def id(self) -> str:
         return f"{self.collection}.{self.name}"
 
+    @pydantic.computed_field
+    def node_sets(self) -> dict:
```

Comment:
```suggestion
    def node_sets(self) -> dict[str, dict[str, Any]]:
```
to fully defined the return type